tools/checkLinks.ts

```diff
@@ -97,13 +97,22 @@ const validLinks = new Map<string, string[]>([
 ]);
 
 // Gather valid links
+const changelogAnchors = [];
 for (const [name, raw] of docFiles) {
   const keyName = `DOCS${name.replaceAll("/", "_").toUpperCase()}`;
   if (!validLinks.has(keyName)) {
     validLinks.set(keyName, []);
   }
   const validAnchors = validLinks.get(keyName)!;
 
+  // The changelog is unique in that each entry acts as it's own potential page,
+  // and each entry can be represented in a single page under it's own H2.
+  // This collects all potential change-log pages, and adds them to the list of
+  // available anchors under `/change_log`.
+  if (name.startsWith("/change_log/")) {
+    changelogAnchors.push(name.split("/")[2].slice(11));
```

Comment:
There are other anchors within the changelog that won't get added to both with just this change.

If the dates are remaining linkable h4s, those won't be added at all currently, and occasionally there are other headers (usually h6) within changelog entries that are traditionally sub anchors of the date (weird but that's how the generator works) that would only be "valid" on the individual page following this logic. Eg: https://discord.com/developers/docs/change-log#july-15-2024-api-updates-since-preview

You're going to need to special case more than just this so I'd suggest storing the result of this if check in `isChangelog` and pushing to `changelogAnchors` alongside `validAnchors` when it's true. Might also need it to properly set the parent anchor (and will the parent anchor be different on the full list vs the individual page?).